packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -218,6 +219,45 @@ export const GCNodeType = {
  */
 export type GCNodeType = (typeof GCNodeType)[keyof typeof GCNodeType];
 
+/**
+ * The contents of a garbage collection sweep message.
+ * @internal
+ */
+export interface ISweepMessageContents {
+	// The ids of nodes that are deleted.
+	deletedNodeIds: string[];
+}
+
+/**
+ * The content type of a garbage collection message content.
+ * @internal
+ */
+export type GarbageCollectionMessageContents = ISweepMessageContents;
+
+/**
+ * The const types of a garbage collection message.
+ * @internal
+ */
+export const GarbageCollectionMessageType = {
+	// Message that is sent after sweep runs.
+	Sweep: "Sweep",
+};
+/**
+ * The types of a garbage collection message.
+ * @internal
+ */
+export type GarbageCollectionMessageType =
```

Comment:
Looks like this resolves to `string`.  See my other comment below for a suggestion on how to approach these types.